src/forms/inputs/textarea.js

```diff
@@ -36,21 +38,29 @@ import { compose, filterInvalidDOMProps, generateInputErrorId } from '../../util
 const propTypes = {
   ...fieldPropTypes,
   hideCharacterCount: PropTypes.bool,
-  maxLength: PropTypes.oneOfType([PropTypes.number, PropTypes.bool])
+  maxLength: PropTypes.oneOfType([PropTypes.number, PropTypes.bool]),
+  forwardedRef: PropTypes.oneOfType([
+    PropTypes.func,
+    PropTypes.shape({
+      current: PropTypes.instanceOf(Element), // eslint-disable-line no-undef
```

Comment:
Typically it's safer to use `// eslint-disable-next-line <rule>` on the line above since eslint and/or prettier can potentially wrap the line when making automated changes

Reply:
I realize that this seems directly contradictory with the precedent set with `meta` below, but I stand by it! 😆 😅

Reply:
Looks like `meta` is actually being used now anyway 😆.